packages/perseus-core/src/accessibility.test.ts

```diff
@@ -182,14 +182,12 @@ describe("isItemAccessible", () => {
 
             expect(isItemAccessible(itemData)).toBe(true);
         });
-    });
 
-    describe("markdown", () => {
-        it("should return false if the item markdown contains any inaccessible images", () => {
+        it("should ignore widgets that are not used in the markdown content", () => {
             const itemData: PerseusItem = {
                 question: {
                     content:
-                        "Here's an image: ![](https://example.com/image.png)",
+                        "Here's an explanation: [[☃ explanation 1]]\n\nAnd a graph: [[☃ interactive-graph 1]]\\But matcher 1 config is unused!!",
```

Comment:
The two widgets in use here are accessible, but `matcher` isn't. It's in the `widgets` dictionary, but is not used in `content` so will never be shown to learners or used in any way.